app_test.go

```diff
@@ -31,13 +38,85 @@ type testCase struct {
 	appConfig *AppConfig
 	testTime  time.Time
 
-	azureUsersSetUp []SourceUser
-	ytUsersSetUp    []YtsaurusUser
-	ytUsersExpected []YtsaurusUser
+	sourceUsersSetUp []SourceUser
+	ytUsersSetUp     []YtsaurusUser
+	ytUsersExpected  []YtsaurusUser
+
+	sourceGroupsSetUp []SourceGroupWithMembers
+	ytGroupsSetUp     []YtsaurusGroupWithMembers
+	ytGroupsExpected  []YtsaurusGroupWithMembers
+}
+
+func getUserID(name string) string {
+	switch name {
+	case aliceName:
+		return "1"
+	case bobName:
+		return "2"
+	case carolName:
+		return "3"
+	}
+	return "4"
+}
```

Comment:
I suppose it would be safer to take a hash from name, because if we have more than 4 users — two of them will get same id and it would be unpleasant tests debugging time.